DataFormats/PatCandidates/interface/Muon.h

```diff
@@ -129,7 +129,9 @@ namespace pat {
     void embedDytMuon();
 
     // add extra timing information
-    void readExtraTimerInfo(const reco::MuonTimeExtra& t);
+    /// 1/beta for prompt particle hypothesis
+    /// (time is constraint to the bunch crossing time)
```

Comment:
please move this comment to the accessor (around L298).
The readTime\* method is here to encapsulate whatever else can be read and the details of 1/beta here are not the most relevant.